docs/overview/04-proof-aggregation/02-handle-valid-proof.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Handle Valid Proof
+---
+
+When a valid proof is routed to a domain via a `domainId` provided in [`submitProof`](../02-mainchain/05-mainchain_api.md#submitproof) the proof is just added to the next aggregation in the given domain and some money are put on hold in the submitter wallet till the aggregation is published. The submitter hold money will be used to pay the aggregation's publisher: see [Publish Attestation](./03-publish-attestations.md) to learn more about the aggregation costs. The proof insertion in the aggregation emits a [`NewProof{statement, domainId, aggregationId}`](../02-mainchain/05-mainchain_api.md#newproof) event and the proof submitter should listen the events to wait a `NewAggregationReceipt{domainId, aggregationId, receipt}` event is emitted to match domain and aggregation: this step is important because the user can compute the Merkle path of its proof in the attestation only by **providing the block where this event was emitted**: the path can be computed by the [`aggregate_statementPath`](../02-mainchain/05-mainchain_api.md#aggregate_statementpath) RPC call.
```

Comment:
```suggestion
When a valid proof is routed to a domain via a `domainId` provided in [`submitProof`](../02-mainchain/05-mainchain_api.md#submitproof) the proof is just added to the next aggregation in the given domain and some money are put on hold in the submitter's wallet till the aggregation is published. The submitter's held money will be used to pay the aggregation's publisher: see [Publish Attestation](./03-publish-attestations.md) to learn more about the aggregation costs. The proof insertion in the aggregation emits a [`NewProof{statement, domainId, aggregationId}`](../02-mainchain/05-mainchain_api.md#newproof) event. The proof submitter should listen to a `NewAggregationReceipt{domainId, aggregationId, receipt}` event containing his domain and aggregation: this step is important because the user can compute the Merkle path of its proof in the attestation only by **providing the block where this event was emitted**: the path can be computed by the [`aggregate_statementPath`](../02-mainchain/05-mainchain_api.md#aggregate_statementpath) RPC call.
```